modules/spring-config/src/main/resources/META-INF/mule-module.xsd

```diff
@@ -87,6 +67,58 @@
         </xsd:complexContent>
     </xsd:complexType>
 
+    <xsd:element name="parameter" type="parameterType">
+        <xsd:annotation>
+            <xsd:documentation>
+                An named element that will be consumed only in the scope of the body element by the list of message processors.
```

Comment:
A parameter element defines an input value for the operation in which it is define. Such parameter must be defined with a meaningful name, a type which defines the kind of content the parameter must have and optionally a default value that will be used if the invocation to the operation does not defines a value for the parameter. The parameter can be accesed within the body definition of the operation using #[..lalalaa..].